crates/runtime/src/derive/combine.rs

```diff
@@ -303,12 +305,15 @@ impl Opened {
             ..
         } = derivation.as_ref().context("missing derivation")?;
 
+        // TODO(johnny): Expose to connector protocol and extract from Open/Opened.
+        let ser_policy = doc::SerPolicy::default();
```

Comment:
Is this where the Rust runtime would configure the `ser_policy`? I.e, if we were actually running container-based connectors using the Rust runtime, the hack to enable this for snowflake and redshift would be here?

Reply:
Almost. This file is specifically about combining for derivations, where I'm not currently aware of a use case for `SerPolicy`, but I also mostly think I lack the required imagination.

There's a corresponding combine.rs i've been working on for captures in a different branch, and materializations will be after that, and yes a hack would live there (once it's written)